projects/core/src/model/cms.model.ts

```diff
@@ -119,6 +119,11 @@ export interface CmsProductCarouselComponent extends CmsComponent {
   scroll?: string;
 }
 
+//configuration controlled through static files
+export interface CmsAddToCartComponent extends CmsComponent {
+  inventoryDisplay?: string;
```

Comment:
https://github.com/SAP/spartacus/pull/11978/files#r661806366

again I don't know what to feel about this as this attribute doesn't exist in the backoffice I believe right? 
All the interface we have as cms components are modeled to the attributes existing in it's corresponding cms component I think.

Reply:
Has been removed